clean_project/construct_dict.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Construct a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import json
+import os
+import yaml
```

Comment:
is this yaml standard library? i don't remember it in it. or maybe you installed the library through pip?